net/adguardhome/files/adguardhome.init

```diff
@@ -4,10 +4,9 @@ PROG=/usr/bin/AdGuardHome
 
 USE_PROCD=1
 
-# starts after network starts
-START=21
-# stops before networking stops
-STOP=89
+# Matches values used by the official AdGuardHome procd init script
+START=95
+STOP=1
```

Comment:
I think we should keep the old stop value. Stopping too early means that if any service send a message over the net on exit for example, there wouldn't be DNS for it to work.